src/components/Navigation.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+const Navigation = () => {
+  return (
+    <nav>
+      <NavLink to="/list" activeClassName="active-link">
```

Comment:
Nice use of activeClassName 👍 

Minor suggestion: you could save this string as a `const` and reuse it in both places. That will be even more useful if you add several more link items!